api-reference/50 Common/Object Structures/PdfCell/padding.md

```diff
@@ -0,0 +1,218 @@
+---
+id: PdfCell.padding
+type: Number | Object
+default: 40
+---
+---
+##### shortDescription
+Generates space around an text of the cell, inside of DataGrid. Uses the measure units, which are specified in the constructor for the [jsPDFDocument](/api-reference/50%20Common/Object%20Structures/PdfExportDataGridProps/jsPDFDocument.md '/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#jsPDFDocument') object.
```

Comment:
Generates space around **the** text of the cell inside **the** DataGrid. Uses the measure units, which are specified in the constructor for the [jsPDFDocument](/api-reference/50%20Common/Object%20Structures/PdfExportDataGridProps/jsPDFDocument.md '/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#jsPDFDocument') object.